src/Tooltip/TooltipBubble.js

```diff
@@ -0,0 +1,89 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import Text from '../Text';
+import StyleSheet from '../PlatformStyleSheet';
+import type { StylePropType } from '../PlatformStyleSheet/StyleTypes';
+
+type Props = {|
+  +arrowHorizontalPosition?: 'flex-start' | 'center' | 'flex-end',
+  +arrowVerticalPosition?: 'flex-start' | 'flex-end',
+  +style?: StylePropType,
+  +text: string,
+  +verticalShift?: number,
+|};
+
+export default function TooltipBubble({
+  arrowHorizontalPosition = 'center',
+  arrowVerticalPosition = 'flex-end',
+  style,
+  text,
+  verticalShift = 3,
+}: Props) {
+  const computedStyles = getComputedStyles(
+    arrowHorizontalPosition,
+    arrowVerticalPosition,
+    verticalShift
+  );
+  return (
+    <View style={[computedStyles.invisibleContainer, style]}>
+      <View style={computedStyles.arrowContainer}>
+        <View style={styles.arrow} />
+      </View>
+      <View style={computedStyles.container}>
+        <Text style={styles.text}>{text}</Text>
+      </View>
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  text: {
+    color: defaultTokens.colorTextWhite,
+    padding: parseFloat(defaultTokens.spaceXSmall),
+    zIndex: 1,
+  },
+  arrow: {
+    backgroundColor: defaultTokens.backgroundBadgeDark,
+    borderRadius: parseFloat(defaultTokens.borderRadiusSmall),
+    height: 17,
+    transform: [{ rotate: '45deg' }],
+    width: 17,
+  },
+});
+
+const getComputedStyles = (
```

Comment:
@jvaclavik Is it a standard way of solving this problem of dynamic styling? If that's performant and taking advantage of `StyleSheet.create`, then I'm all for it!

Reply:
But I think this is the point of using `StyleSheet.create`, it sort of does that for you as far as I know. Apparently, you cannot notice the difference at first, but if an entire codebase did styles without it, it would become a huge burden on resources.